src/nope.ts

```diff
@@ -0,0 +1,61 @@
+import { appendErrors, transformToNestObject, Resolver } from 'react-hook-form';
+import NopeObject from 'nope-validator/lib/umd/NopeObject';
+
+const parseNopeSchema = (error: any, validateAllFieldCriteria: boolean) => {
+  const errors = Object.keys(error);
+
+  return Array.isArray(errors)
```

Comment:
I think `errors` is always `Array` type. Is it wrong?

Reply:
Nope! This is the signature:

```ts
export interface ShapeErrors {
    [key: string]: string;
}
```